### ISSUE TITLE
gaussian parameters

### ISSUE THREAD

**Issue by ChunmianLin** [xuannianz/keras-GaussianYOLOv3#2]
Hi, I want to know where does the introduction of gaussian parameters for uncertainty calculation?
Thanks.

**Comment by xuannianz** [comment]
https://github.com/xuannianz/keras-GaussianYOLOv3/blob/b5c321e74f3d1faf37e6e8a8226e5cd2101dfa46/model.py#L115-L121
Here you can see there are `num_classes + 9` params for each anchor instead of `num_classes + 5` in original YOLOv3.
The first 8 params are ![](http://latex.codecogs.com/gif.latex?\mu)<sub>tx</sub> ![](http://latex.codecogs.com/gif.latex?\mu)<sub>ty</sub> ![](http://latex.codecogs.com/gif.latex?\mu)<sub>tw</sub> ![](http://latex.codecogs.com/gif.latex?\mu)<sub>th</sub> ![](http://latex.codecogs.com/gif.latex?\sigma)<sub>tx</sub> ![](http://latex.codecogs.com/gif.latex?\sigma)<sub>ty</sub> ![](http://latex.codecogs.com/gif.latex?\sigma)<sub>tw</sub> ![](http://latex.codecogs.com/gif.latex?\sigma)<sub>th</sub>.

**Comment by ChunmianLin** [comment]
Thanks. 
But I still confused that how an where do you define the gaussian distribution and use its parameters for uncertainty prediction?

**Comment by xuannianz** [comment]
The gaussian distribution expression is defined in loss function.

**Comment by ChunmianLin** [comment]
Sorry, I still cannot localize the gaussian definition, and don't understanding how it work for uncertainty calculation. Could you please tell me in detail?
Thanks.

**Comment by linchunmian** [comment]
Thanks. 
If I want to test model flops and FPS, have your code provide these computations?

发送自 Windows 10 版邮件应用

发件人: xuannianz
发送时间: 2019年11月20日 8:54
收件人: xuannianz/keras-GaussianYOLOv3
抄送: ChunmianLin; Author
主题: [SPAM] Re: [xuannianz/keras-GaussianYOLOv3] gaussianparameters (#2)

The gaussian distribution expression is defined in loss function.
